test/jdk/javax/swing/JInternalFrame/8146321/JInternalFrameIconTest.java

```diff
@@ -61,7 +63,7 @@
 
     public static void main(String[] args) throws Exception {
         robot = new Robot();
-        robot.delay(2000);
+        robot.setAutoDelay(100);
```

Comment:
Why do you need an autodeley here? It seems the test does not use any of mouseXXX/keyXXX methods.